js/packages/teams-ai/src/models/OpenAIModel.ts

```diff
@@ -341,10 +345,19 @@ export class OpenAIModel implements PromptCompletionModel {
                     if (delta.content) {
                         message.content += delta.content;
                     }
-                    // TODO: handle tool calls
-                    // if (delta.tool_calls) {
-                    //     message.tool_calls = delta.tool_calls;
-                    // }
+                    // Handle tool calls
```

Comment:
maybe we should add a TODO since steve mentioned streaming requires some changes first to enable tools

Reply:
This TODO was specifically for the changes I needed to make, and I'm not sure where exactly I would put the todo's for his work, so I'll leave as-is.